content/en/users/data/management/rucio/_index.md

```diff
@@ -1,31 +1,30 @@
 ---
-title: "Multi-VO Rucio for data management"
+title: Rucio
+linkTitle: Rucio
 type: docs
-linkTitle: "Rucio"
-weight: 120
+weight: 10
 description: >-
-  Multi-VO Rucio Documentation for new users to follow to get access to, and use
-  Rucio.
+  Organise and access data at scale with Rucio
 ---
 
-## Overview
+## What is it?
 
 Built on more than a decade of experience in LHC experiments, Rucio serves the
 data needs of any modern scientific experiments. Rucio can manage large amounts
 of data, countless numbers of files, heterogeneous storage systems, globally
-distributed data centres, monitoring and analytics.
+distributed data centres, with monitoring and analytics.
 
-Rucio allows you to manage your data with expressive statements. You to say what
-you want, and let Rucio figure out the details of how to do it. For example,
-Three copies of my file on different continents with a backup on tape. You can
+Rucio **allows management of data with expressive statements**. You to say what
+you want, and Rucio will figure out the details of how to do it. For example,
+three copies of my file on different continents with a backup on tape. You can
 also automatically remove copies of data after a set period or once its access
 popularity drops.
 
 While Rucio is extremely scalable, the STFC Rucio Data Management Service is
 designed for smaller communities, with expected data needs up to tens of
 Petabytes. The fact that the underlying Rucio infrastructure is managed by STFC,
-allows communities to easily start using and or test Rucio with little setup
-cost required.
+allows communities to easily start using and/or test Rucio with little setup
+cost.
 
 ## Official Rucio Pages
```

Comment:
Going back to the DataHub comment: this is similar to the "links" in DataHub, no? Maybe doing the same for DataHub, move the "Links" content to the overview DataHub page?

Reply:
Yep. These should be be compacted to the absolute minimum, and the result either put as notes or as links on terms in the text. Future improvement, maybe not in this PR.